demos/keyboard_nav/index.html

```diff
@@ -22,28 +22,32 @@
   <h1><a href="https://developers.google.com/blockly/">Blockly</a> &gt;
     <a href="../index.html">Demos</a> &gt; Keyboard Navigation</h1>
 
-  <p>You can enter keyboard navigation mode by <b>shift clicking anywhere on the
-  workspace or on a block.</b> <br>
-
-  You can then use the WASD keys to move around the workspace or click the show
-  traversal button to display an alternative way to navigate the workspace.<br>
-
-  The demo uses <a href="https://en.wikipedia.org/wiki/Tree_traversal#Pre-order_(NLR)">pre
-  order tree traversal</a> as an alternative way to navigate the blocks, connections,
-  and fields on the workspace.<br><br>
+  <p>Keyboard Navigation is our first step towards an accessible Blockly.<br>
+  You can enter accessibility mode by <b>shift clicking anywhere on the
+  workspace or on a block.</b> <br>Some basic commands for moving around are below
+  but for a complete list of commands please visit here(Documentation on its way!).<br><br>
```

Comment:
Lets not claim there's a complete list of commands somewhere before we have a link to it. Either leave that off or say something like "More complete documentation is still in progress."